frontend/src/fetchers/__tests__/activityReports.js

```diff
@@ -21,10 +21,11 @@ describe('activityReports fetcher', () => {
         sortDir: 'desc',
         offset: 0,
         limit: 10,
+        filters: 'filters',
       };
 
       fetchMock.get(join('api', 'activity-reports'), [], { query });
-      await getReports();
+      await getReports(undefined, undefined, undefined, undefined, 'filters=filters');
```

Comment:
Does this error without the `filters=filters` at the end?

Reply:
Test still pass, added it since it helped increase the test coverage. I can create a seperate test w/o the `filter` part

Reply:
Oh I get it now, it is probably fine as is.